crates/pop-cli/src/commands/call/chain.rs

```diff
@@ -615,7 +615,7 @@ mod tests {
 	use url::Url;
 
 	const BOB_SURI: &str = "//Bob";
-	const POP_NETWORK_TESTNET_URL: &str = "wss://rpc1.paseo.popnetwork.xyz";
+	const POP_NETWORK_TESTNET_URL: &str = "wss://rpc2.paseo.popnetwork.xyz";
```

Comment:
Should be separate PR imo

Reply:
Technically, you're correct,  but I considered it simple enough to introduce here and avoid unnecessary delays.